transcom/mymove
# B 17664 remove vendor panel on ppm shipments

## B-17664 remove vendor panel on ppm shipments

## Summary

"As a TOO when creating a MTO for a Partial PPM

I should not see an option of selecting a type of “Vendor” (either GHC Prime Contractor or External vendor)

So that as the TOO I can enter the Origin info and complete the PPM without this unnecessary step"

Previously this panel only appeared on the page if the user is a TOO AND the shipment type is NOT HHG.

Since this task requires removing the panel for TOO when editing a PPM only, I have added a clause that will not show this panel for PPM moves.

This means that the new case for the panel showing will be: The user is a TOO AND the shipment type is NOT HGG AND the shipment type is NOT PPM. If all 3 conditions are not met, the panel will not show.

Now, the panel will only show for NTS or NTSR shipment options when viewing as a TOO. A quick adjustment to my local tests confirm this.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access milmove's office client
2. Login as a TOO role
3. Look for existing shipments.
4. Select one, and then select "Edit Shipment"
5. See if the Vendor panel is present, test the form by submitting it.

### Frontend


### Backend


## Screenshots
<img width="682" alt="Screenshot 2023-11-14 at 10 27 56 AM" src="https://github.com/transcom/mymove/assets/148906585/385309ee-637d-4837-8935-741c2b5e18f6">
<img width="1364" alt="Screenshot 2023-11-14 at 10 28 24 AM" src="https://github.com/transcom/mymove/assets/148906585/42f39e96-456e-4ae7-87e4-c50dc553083b">
